Deamon - "mmmagic" installation failed due to permission error

Open again (reason https://github.com/VilhelmPrytz/pterodactyl-installer/issues/7#issuecomment-589310629)
Currently the installation of Deamon on Debian 10 / Ubuntu 19.X does not work.
"npm install --only=production always" runs into several errors.
nodejs -v -> v13.9.0
npm -v -> 6.13.7

Debian 9 with nodejs 10.X however works.
The installation script works perfectly, the problem is clearly due to pterodactyl.

**Describe the bug**
Here the Error-Log:
```
10186 verbose stack Error: mmmagic@0.5.0 install: `node-gyp rebuild`
10186 verbose stack Exit status 1
10186 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
10186 verbose stack     at EventEmitter.emit (events.js:321:20)
10186 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
10186 verbose stack     at ChildProcess.emit (events.js:321:20)
10186 verbose stack     at maybeClose (internal/child_process.js:1026:16)
10186 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
10187 verbose pkgid mmmagic@0.5.0
10188 verbose cwd /srv/daemon
10189 verbose Linux 4.19.0-8-amd64
10190 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "--only=production"
10191 verbose node v13.9.0
10192 verbose npm  v6.13.7
10193 error code ELIFECYCLE
10194 error errno 1
10195 error mmmagic@0.5.0 install: `node-gyp rebuild`
10195 error Exit status 1
10196 error Failed at the mmmagic@0.5.0 install script.
10196 error This is probably not a problem with npm. There is likely additional logging output above.
10197 verbose exit [ 1, true ]`

And the log of the open ssh-session:
`make: *** [magic.target.mk:122: Release/obj.target/magic/src/binding.o] Fehler 1
make: Verzeichnis „/srv/daemon/node_modules/mmmagic/build“ wird verlassen
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:321:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Linux 4.19.0-8-amd64
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /srv/daemon/node_modules/mmmagic
gyp ERR! node -v v13.9.0
gyp ERR! node-gyp -v v5.0.7
gyp ERR! not ok`
...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mmmagic@0.5.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the mmmagic@0.5.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-02-19T19_53_34_839Z-debug.log`
```

Tested with:
- Debian 9
=> NodeJS 10: Working
=> NodeJS 12 and 13: Not Working
- Debian 10
=> Every NodeJS-version not working
- Ubuntu 19.04
=> Every NodeJS-version not working
- Ubuntu 19.10
=> Every NodeJS-version not working

I get this error on start-up:

`> pteronode@0.6.12 start /srv/daemon
> node src/index.js | node_modules/bunyan/bin/bunyan -o short

18:41:56.559Z  INFO wings: + ------------------------------------ +
18:41:56.560Z  INFO wings: | Running Pterodactyl Daemon v0.6.12    |
18:41:56.560Z  INFO wings: |        https://pterodactyl.io        |
18:41:56.560Z  INFO wings: |  Copyright 2015 - 2019 Dane Everitt  |
18:41:56.560Z  INFO wings: + ------------------------------------ +
18:41:56.560Z  INFO wings: Loading modules, this could take a few seconds.
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module 'mmmagic'
Require stack:
- /srv/daemon/src/controllers/fs.js
- /srv/daemon/src/controllers/server.js
- /srv/daemon/node_modules/rfr/lib/rfr.js
- /srv/daemon/node_modules/rfr/index.js
- /srv/daemon/src/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/srv/daemon/src/controllers/fs.js:30:13)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/srv/daemon/src/controllers/fs.js',
    '/srv/daemon/src/controllers/server.js',
    '/srv/daemon/node_modules/rfr/lib/rfr.js',
    '/srv/daemon/node_modules/rfr/index.js',
    '/srv/daemon/src/index.js'
  ]
}`

--- a/install-daemon.sh
+++ b/install-daemon.sh
@@ -293,7 +293,7 @@ function ptdl_dl {
   cd /srv/daemon || exit
 
   curl -L "$DL_URL" | tar --strip-components=1 -xzv
-  npm install --only=production
+  npm install --only=production --unsafe-perm
 
   echo "* Done."
 }

